config/config.properties

```diff
@@ -29,82 +29,82 @@
 # processing reads, background maintenance operations and background operation processing.
 # Valid values: Positive integer.
 # Recommended setting: 2 * Number of containers per node, minimum 20.
-#pravegaservice.threadPoolSize=30
+#pravegaservice.threadPool.size=30
 
 # Maximum number of threads in the Thread Pool used for Tier2 tasks (reading, writing, create, delete, etc.).
 # Valid values: Positive integer.
 # Recommended setting: reasonably large number which does not cause thrashing, minimum 20, recommended value 200.
-#pravegaservice.storageThreadPoolSize=200
+#pravegaservice.storageThreadPool.size=200
 
 # TCP port where the SegmentStore will be listening for incoming requests.
 # Valid values: Positive integer in the valid TCP port ranges.
-pravegaservice.listeningPort=12345
+pravegaservice.service.listener.port=12345
 
 # Full URL (host:port) where to find a ZooKeeper that can be used for coordinating this Pravega Cluster.
 # Required.
-pravegaservice.zkURL=localhost:2181
+pravegaservice.zk.connect.uri=localhost:2181
 
 # Number of milliseconds to wait between ZooKeeper retries.
 # Valid values: Positive integer.
 # Recommended values: Between 5000 and 10000 (too high will cause failover delays).
-#pravegaservice.zkRetrySleepMs=5000
+#pravegaservice.zk.connect.retries.interval.milliseconds=5000
 
 # Number of times to retry a failed attempt to contact ZooKeeper.
 # Valid values: Positive integer.
 # Recommended values: Between 3 and 5.
-#pravegaservice.zkRetryCount=5
+#pravegaservice.zk.connect.retries.count.max=5
 
 # Session timeout for ZooKeeper.
 # Valid values: Positive integer.
-#pravegaservice.zkSessionTimeoutMs=10000
+#pravegaservice.zk.connect.sessionTimeout.milliseconds=10000
 
 # Enable security for the ZK client.
-#pravegaservice.secureZK=false
+#pravegaservice.zk.connect.security.enable=false
 
 # Location of trust store for secure connection to ZK.
-#pravegaservice.zkTrustStore=
+#pravegaservice.zk.connect.security.tls.trustStore.location=
 
 #Location of password file to access the trust store.
-#pravegaservice.zkTrustStorePasswordPath=
+#pravegaservice.zk.connect.security.tls.trustStore.pwd.location=
 
 # DataLog implementation for Tier 1 storage.
 # Valid values: BOOKKEEPER, INMEMORY.
 # Default value: BOOKKEEPER
-pravegaservice.dataLogImplementation=BOOKKEEPER
+pravegaservice.dataLog.impl.name=BOOKKEEPER
 
 # Storage implementation for Tier 2 storage.
 # Valid values: HDFS, FILESYSTEM, EXTENDEDS3, INMEMORY.
 # Default value: HDFS
-# pravegaservice.storageImplementation=HDFS
+# pravegaservice.storage.impl.name=HDFS
 
 # Storage NO-OP Mode: in No-Op mode, user stream segment writing is no-oped; user stream segment reading is not supported.
 # This mode is used to avoid storage interference in testing while still keep the system functioning as usual.
 # NOTE: pravegaservice.storageImplementation is still used to store metadata and system segments, which are required for the functioning of the Pravega Cluster.
```

Comment:
Did you fix this reference? There may be others, so please check this file for references in the comments.

Reply:
Fixed.